lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -170,6 +174,94 @@ SessionState::SessionState(
       static_rules_(rule_store),
       credit_map_(4, &ccHash, &ccEqual) {}
 
+/*For 5G which doesn't have response context*/
+SessionState::SessionState(
+    const std::string& imsi, const std::string& session_ctx_id,
+    const SessionConfig& cfg, StaticRuleStore& rule_store)
+    : imsi_(imsi),
+      session_id_(session_ctx_id),
+      // Request number set to 1, because request 0 is INIT call
+      request_number_(1),
+      /*current state would be CREATING and version would be 0 */
+      curr_state_(CREATING),
+      config_(cfg),
+      current_version_(0),
+      static_rules_(rule_store) {}
+
+/* get-set methods of new messages  for 5G*/
+uint32_t SessionState::get_current_version() {
+  return current_version_;
+}
+
+void SessionState::set_current_version(int new_session_version) {
+   current_version_ = new_session_version;
+}
+/* Add PDR rule to this rules session list */
+void SessionState::insert_with_static_rules(SetGroupPDR *rule) {
```

Comment:
Probably clearer names would be `insert_pdr`, `insert_far`, etc.

Reply:
changed as "current_version"

Reply:
Changed

Reply:
changed